### DISCUSSION TITLE
Issues with npm run start

### DISCUSSION THREAD

**Discussion by rishiskhare** [community/community#84743] [npm]
### Body

I forked this repo: https://github.com/AXeL-dev/distract-me-not

I installed all dependencies. However, I keep getting this error when running npm run start:
![image](https://github.com/community/community/assets/30673002/d8046afd-e658-47cf-a7d6-d192a60a27be)

Does anyone know how to resolve this? I've tried reinstalling Node and reinstalling dependencies, all with no luck.

Note: I've posted this on the repo's discussions as well, but this repo hasn't had an update in over six months, so posting here to see if it can be resolved quicker.

### Guidelines

- [x] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Mehdi-RaTo** [comment]
Hi. Test it by Node.js LTS version. for example: 20.10.0 LTS

**Comment by LuksJobs** [comment]
Quick fix: Run this in a terminal inside your project.

`export NODE_OPTIONS=--openssl-legacy-provider
`

So if you'd like to continue using the NPM script, you can change the build script to:

// package.json

```
"scripts": {
    "start": "react-scripts --openssl-legacy-provider start",
    "build": "react-scripts --openssl-legacy-provider build",
}
```

**Comment by abde-r** [comment]
Try to update the node modules directory by
`> rm -rf node_modules`
`> npm update`

**Comment by kmadaminjanova0508** [comment]
try npm start or npm run dev instead O check node version
